pkg/services/authz/rbac/mapper.go

```diff
@@ -63,6 +63,10 @@ func newMapper() mapper {
 		"iam.grafana.app": {
 			"teams": newResourceTranslation("teams", "id", false),
 		},
+		"secret.grafana.app": {
+			"securevalues": newResourceTranslation("secrets-manager.securevalues", "name", false),
```

Comment:
@kalleep @gamab Thoughts on having the resource being `secrets-manager.securevalues` and not just `securevalues`? Any best practices around this?

Reply:
What do your prefer? :slightly_smiling_face: 
I personally lean towards the prefixed approach. It clarifies the context of the resource, which is particularly useful for `keepers` imo.

In the code base we have core elements `datasources`, `folders`, `teams`, and the `.` usually indicates a sub resource/functionality `folders.permissions`, `teams.roles`, `datasources.caching`.

For alerting related permissions, they are all prefixed, making it obvious that rules, notifications etc... are alerting resources: `alert.rules`, `alert.notifications`, `alert.silences`.

Reply:
You could shorten to `secrets.securevalues` I think though :)

Reply:
I think you can skip `secrets-manager` prefix if it don't give us any value or collide with another action but its up to you

Reply:
Awesome, went with `secret` as a prefix since the app is `secret.grafana.app` and we are deploying to `grafana-secret` namespace 🤷 😄